CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ the detailed section referring to by linking pull requests or issues.
 #### Minor
 
```

Comment:
Compared to before, in my opinion the status is too prominent. They are supposed to be rather a side note, imo.

I experimented with moving it to the right at least, but we still have some discrepancies between the detail dialog and the status in the table.

I believe in an ideal world we would do a two-row row, but since we are using the Material Table we have to see how we do it

Reply:
I changed the text color and made the Icon and font size smaller as well which helps remove some of the prominence.